sdk/storage/Azure.Storage.Blobs/tests/ClientSideEncryptionTests.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.Cryptography;
+using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Keys.Cryptography;
+using Azure.Storage.Cryptography;
+using Azure.Storage.Cryptography.Models;
+using Azure.Storage.Test.Shared;
+using Azure.Storage.Tests.Shared;
+using NUnit.Framework;
+
+namespace Azure.Storage.Blobs.Test
+{
+    public class ClientSideEncryptionTests : BlobTestBase
+    {
+        private const string ThrowawayAlgorithmName = "blah";
+        public ClientSideEncryptionTests(bool async, BlobClientOptions.ServiceVersion serviceVersion)
+            : base(async, serviceVersion, null /* RecordedTestMode.Record /* to re-record */)
+        {
+        }
+
+
+        #region Utility
+
+        private byte[] LocalManualEncryption(byte[] data, byte[] key, byte[] iv)
```

Comment:
nit: method naming should be verb-ish. `EncryptLocally` or `DoLocalEncryption`. noun-ish terms are reserved for types. btw. what does manual mean?

Reply:
It means I'm invoking the encryption myself, instead of going through the client's automated processes. Maybe explicit is a better word than manual?

Reply:
Maybe bypass may be a candidate term to consider? Is that what you're trying to do?

Reply:
`Independent`/`External`/`Baseline`? 

Manual indicates human involvement in the process.